### ISSUE TITLE
Exchange section

### ISSUE THREAD

**Issue by BennyOptions** [balancednetwork/balanced-network-info#141]
Seems that when you set the date to a time prior to any POL for that asset, it returns something a bit weird. It should be 0/0 I would think. Talking about the column furtherest to the right.

Also, while I'm here, @parrot9design I'd say that there could be a better descriptor than "EXCHANGE", though I understand where you're coming from. I'd recommend Protocol Owned Liquidity, as anybody across the industry from any ecosystem will recognize this. Hoping new users/potential investors immediately understand what this is w/o any friction/confusion. I don't necessarily love every new piece of lingo that comes out of the industry, but I think there are tangible benefits to making this a recognizable term for the balanced target market.

<img width="1196" alt="Screen Shot 2023-02-03 at 6 02 12 PM" src="https://user-images.githubusercontent.com/43685868/216588150-8afc5420-5e15-4c11-998e-54810b7d2ae5.png">

**Comment by BennyOptions** [comment]
UPDATE: This is only a design issue. The issue mentioned above regarding amount of POL was accurate. It was a small test amount prior to making the large deposit.

**Comment by parrot9design** [comment]
We started out with `Protocol-owned liquidity`, but the length was an issue as it overcomplicated the section and is difficult to read in all-caps, so we decided it would better to use a shorter alternative (the fact that it’s jargon was a tertiary issue).

`Protocol-owned` is actually redundant to specify in a `Holdings` section, as _all_ the assets listed are protocol-owned. `Liquidity` doesn’t work well on its own, so `Exchange` was the only heading that made sense, and is consistent with the main Stats page. Was also considering its use in a sentence "How much value does the Balanced DAO hold?" "It holds $x in the DAO Fund and $x on the exchange..."

If you have any other suggestions I’m happy to consider them, but `Protocol-owned liquidity` didn't seem like the right choice here.
